packages/dd-trace/src/priority_sampler.js

```diff
@@ -185,7 +254,13 @@ class PrioritySampler {
     return allowed
   }
 
-  _getPriorityByAgent (context) {
+  /**
+   *
+   * @param context {DatadogSpanContext}
+   * @returns {SamplingPriority}
+   * @private
+   */
+  #_getPriorityByAgent (context) {
```

Comment:
```suggestion
  #getPriorityByAgent (context) {
```

And also the other entries :)

The underscore is a legacy way where private properties didn't exist to "mark" something as "internal".